src/api/modelTypes.js

```diff
@@ -486,7 +486,7 @@ export type Submessage = $ReadOnly<{|
  * See also `MessagesState` for discussion of how we fetch and store message
  * data.
  */
-export type Message = $ReadOnly<{|
+type MessageBase = $ReadOnly<{|
```

Comment:
6d6ce7720 modelTypes: Start separating `PmMessage` and `StreamMessage`.

(The jsdoc should probably also be moved/refactored, somewhere around this commit.)

Reply:
Yeah. Probably it should simply move to the new `Message` definition -- that's what will cause it to appear when looking in the IDE at uses of `Message`.

And then at this spot, and at `PmMessage` and `StreamMessage`, perhaps add just a few words pointing at `Message`.